README.md

```diff
@@ -24,3 +24,14 @@ pip install -r requirements.txt
 Build all: `./scripts/render-all.sh`
 
 Lint filenames: `./scripts/lint-filenames.py`
+
+## How to SR(A)WN
+
+1. Write an entry in `SRYYYY/YYYY-MM-DD-srawn-NN.md`, substituting the programme year, calendar year, day, month, and issue number within the programme year (e.g. `SR2024/2023-11-05-srawn-03.md`).
+2. Merge into `main`. This automatically updates the website.
+3. Locally, run `scripts/render-all.sh`. This will generate the HTML email in `out/html/SRYYYY/2023-11-05-srawn-03.html`.
```

Comment:
Presumably:
- or visit the website
- or download the artefact and view that

Reply:
Went for "visit the website" 👍